corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -328,16 +328,26 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
         },
 
         initialize: function (options) {
-            this.styles = options.styles;
-            this.hasNoItems = options.collection.length === 0;
-            this.redoLast = options.redoLast;
+            var self = this;
+            self.styles = options.styles;
+            self.hasNoItems = options.collection.length === 0;
+            self.redoLast = options.redoLast;
             if (sessionStorage.selectedValues !== undefined) {
                 let parsedSelectedValues = JSON.parse(sessionStorage.selectedValues)[sessionStorage.queryKey];
-                this.selectedCaseIds = parsedSelectedValues !== undefined && parsedSelectedValues !== '' ? parsedSelectedValues.split(',') : [];
+                self.selectedCaseIds = parsedSelectedValues !== undefined && parsedSelectedValues !== '' ? parsedSelectedValues.split(',') : [];
             } else {
-                this.selectedCaseIds = [];
+                self.selectedCaseIds = [];
             }
-            this.isMultiSelect = options.isMultiSelect;
+            self.isMultiSelect = options.isMultiSelect;
+
+            FormplayerFrontend.on("multiSelect:updateCases", function (action, caseIds) {
```

Comment:
I found the same thing as @shubham1g5  with the disabled continue button https://github.com/dimagi/commcare-hq/pull/32240#issuecomment-1275902951 - specifically when using case search - and seemed to trace it back to this line. 
I can't say I fully understand what the line does, but reverting to its previous version seemed to resolve it:
```javascript
FormplayerFrontend.off("multiSelect:updateCases").on("multiSelect:updateCases", function (action, caseIds) {
```